docs/cvl/methods.md

```diff
@@ -307,12 +316,72 @@ Contract methods can also be summarized using CVL {doc}`functions` or
 {ref}`ghost-axioms` as approximations.  Contract calls to the summarized method
 are replaced by calls to the specified CVL functions.
 
+There are a few restrictions on the functions that can be used as approximations:
+ - Functions used as summaries are not allowed to call contract functions.
+ - Functions used as summaries may not have accept arguments or return values that have struct or array types.
+
 To use a CVL function or ghost as a summary, use a call to the function in
-place of the summary type.  The function call can only refer directly to the
+place of the summary type.  The function call can refer to the
 variables defined as arguments in the summary declarations; expressions
 that combine those variables are not supported.
 
-There are a few restrictions on the functions that can be used as approximations:
- - Functions used as summaries are not allowed to call contract functions.
- - Functions used as summaries may not have accept arguments or return values that have struct or array types.
+The function call may also use the special variable `calledContract`, which
+gives the address of the contract on which the summarized method was called.
+This is useful for identifying the called contract in {ref}`wildcard summaries
+<cvl2-wildcards>`.
+
```

Comment:
should we state the calledContract can only be used with the summarization expression and not in any other cvl statement?